tests/phpunit/tests/functions/wpRefererFeld.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * @group functions.php
+ * @covers ::wp_referer_field
+ */
+class Tests_Functions_wpRefererField extends WP_UnitTestCase {
+
+	/**
+	 * @ticket 54106
+	 */
+	public function test_wp_referer_field() {
+		ob_start();
+		wp_referer_field();
+		$content = ob_get_clean();
+
+		$this->assertSame( '<input type="hidden" name="_wp_http_referer" value="" />', $content );
```

Comment:
Use `get_echo` function here.

```suggestion
		$this->assertSame( '<input type="hidden" name="_wp_http_referer" value="" />', get_echo( 'wp_referer_field' ) );
```